lib/src/main/java/com/shopify/checkoutsheetkit/CheckoutWebView.kt

```diff
@@ -165,10 +164,13 @@ internal class CheckoutWebView(context: Context, attributeSet: AttributeSet? = n
         override fun onRenderProcessGone(view: WebView, detail: RenderProcessGoneDetail): Boolean {
             return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O && !detail.didCrash()) {
                 // Renderer was killed because system ran out of memory.
-
-                // Removing the view from `CheckoutWebViewContainer will trigger a cache clear
-                // and call webView.destroy()
-                (view.parent as ViewGroup).removeView(view)
```

Comment:
Is this happening somewhere else now?

Reply:
Yeah it is, although a little more indirectly.

onCheckoutViewFailedWithError -> closeCheckoutDialogWithError -> dismiss -> removeFromContainer -> clearCache.